### PR TITLE
chore: add execution API disclaimer to docs

### PR DESCRIPTION
**Pull request by MSevey**
## Overview

To help us bridge the gap between updating the execution tutorials and the execution API updates, I thought it would be a good idea to add a disclaimer. 

## Release Notes

- **New Features**
  - Introduced the `ExecutionCallout` component to inform users about updates to the Rollkit execution system and encourage feedback.
  - Enhanced tutorials with visual callouts for tips and warnings, improving user guidance.

- **Documentation Improvements**
  - Updated tutorial links for better organization under the "Execution" category.
  - Reorganized content in multiple tutorials for clarity, including detailed instructions for setup and configuration.
  - Added new sections in tutorials to provide contextual tips and warnings, enhancing user experience.

These changes enhance user experience by providing clearer instructions and improving the overall structure of the documentation.

### CODE REVIEW

File: .vitepress/components/execution_callout.vue [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<template>
+    <div>
+        <p>Rollkit is currently undergoing an execution overall with the
```
Comment (tzdybal) [inline .vitepress/components/execution_callout.vue:3]: Overhaul?

File: .vitepress/config.ts [hunk 2]
```diff
@@ -275,20 +275,20 @@ function sidebarHome() {
...
           items: [
-            { text: "CosmWasm", link: "/tutorials/cosmwasm" },
+            { text: "CosmWasm", link: "/tutorials/execution/cosmwasm" },
             {
               text: "EVM",
               collapsed: true,
               items: [
-                { text: "Omni Octane EVM", link: "/tutorials/octane-evm" },
-                { text: "BeaconKit EVM", link: "/tutorials/beaconkit" },
+                { text: "Omni Octane EVM", link: "/tutorials/execution/octane-evm" },
```
Comment (yarikbratashchuk) [inline .vitepress/config.ts:283]: do we still need Omni?

**Review by yarikbratashchuk — APPROVED** [review]

**Review by tuxcanfly — APPROVED** [review]
OK.

File: .vitepress/config.ts (hunk 2 above)
Comment (MSevey) [reply .vitepress/config.ts:283]: not sure, do we? @Manav-Aggarwal

File: .vitepress/components/execution_callout.vue (hunk 1 above)
Comment (MSevey) [inline .vitepress/components/execution_callout.vue:3]:
```suggestion
        <p>Rollkit is currently undergoing an execution overhaul with the
```

**Review by jcstein — APPROVED** [review]
utACK

**Pull request merged**